clients/translate/lib/google_api/translate/v2/api/detections.ex

```diff
@@ -31,19 +31,19 @@ defmodule GoogleApi.Translate.V2.Api.Detections do
 
   - connection (GoogleApi.Translate.V2.Connection): Connection to server
   - opts (KeywordList): [optional] Optional parameters
-    - :oauth_token (String.t): OAuth 2.0 token for the current user.
```

Comment:
Filed an issue for the parameter ordering: https://github.com/GoogleCloudPlatform/elixir-google-api/issues/57